libs/storage/Tsavorite/cs/src/core/Index/Common/LogSizeTracker.cs

```diff
@@ -151,16 +152,16 @@ public void IncrementSize(long size)
         /// Performs resizing by waiting for an event that is signaled whenever memory utilization changes.
         /// This is invoked on the threadpool to avoid blocking calling threads during the resize operation.
         /// </summary>
-        async Task ResizerTask()
+        async Task ResizerTask(CancellationToken token)
         {
-            while (true)
+            while (!token.IsCancellationRequested)
```

Comment:
when we leave the while loop, we usually set some variable, that the main dispose can check to make sure dispose is completed. i believe this is done in a few other places.

Reply:
Updated to have the StoreWrapper dispose method wait for the resizer task cancellation to complete.